src/Imath/ImathRandom.h

```diff
@@ -130,35 +130,35 @@ inline Rand32::Rand32 (unsigned long int seed)
     init (seed);
```

Comment:
IMATH_HOSTDEVICE here, too? (Line 128)

Reply:
thanks, good catch. I was pretty bleary-eyed at the end.